### DISCUSSION TITLE
能给拓展倍速加一个一键改变（原倍速/记忆倍速状态）的按键吗

### DISCUSSION THREAD

**Discussion by Clov3rBug** [the1812/Bilibili-Evolved#4525] [功能建议 / Ideas]
就是平时使用记忆倍速的功能比较多，经常需要更改（记忆倍速/原倍速）默认的鼠标停留再选择效率有点低，可以像是telegram一样，直接点一下倍速键就可以更改这两种状态吗（记忆倍速/原倍速），如果有开记忆倍速插件的就默认记忆倍速和原倍速切换，没有开这个插件的就默认2倍速或者1.5倍速什么的

装了快捷键拓展的话可以shift+/ 快速切换，我平时也一直在用，但是如果再加上这个功能的话就可以不用到键盘直接切换了

**Comment by miku1958** [comment]
我希望可以拓展一下切换倍速的功能, 让快捷键支持单个按键可以切换1倍数(或者自定义的其他倍速), 比如我自己很多播放器都是c增加, x减少, z恢复1倍数
